Langs/README.md

```diff
@@ -0,0 +1,38 @@
+# Guidelines
+
+## Requirements
+
+- **Find Your Language Code Here:** https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm
+- Always use the latest template (0000.lang)
```

Comment:
Missing dot here, either use them or not, be consistent.

Reply:
Place a dot there.